src/useTimeout/useTimeout.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useRef } from 'react'
+
+export function useTimeout (callback: Function, delay: number) {
+  const savedCallback = useRef<Function>()
```

Comment:
TIL `useRef` can be useful outside the context of DOM nodes.
📚 https://reactjs.org/docs/hooks-reference.html#useref